### ISSUE TITLE
World SQL

### ISSUE THREAD

**Issue by Nephelion** [Arkania/ArkCORE-NG#257]
Hi 
sorry i use issue for my problem 
i dot know where i can find [World.sql] TDB ...
start world sql [ clean world ]

**Comment by gpn39f** [comment]
you can find link to snapshoot on our forum: http://gpn39f.spdns.de/forum/

**Comment by Nephelion** [comment]
@gpn39f   
error on import world snapshot
and me download curent snapshot in folders..thats right ?
https://pastebin.com/9SZ8W33A

Query:
CREATE TABLE `game_event` (
  `eventEntry` tinyint(3) unsigned NOT NULL COMMENT 'Entry of the game event',
  `start_time` timestamp NOT NULL DEFAULT '1970-01-01 01:00:01' COMMENT 'Absolute start date, the event will never start before',
  `end_time` timestamp NOT NULL DEFAULT '1970-01-01 01:00:01' COMMENT 'Absolute end date, the event will never start afler',
  `occurence` bigint(20) unsigned NOT NULL DEFAULT '5184000' COMMENT 'Delay in minutes between occurences of the event',
  `length` bigint(20) unsigned NOT NULL DEFAULT '2592000' COMMENT 'Length in minutes of the event',
  `holiday` mediumint(8) unsigned NOT NULL DEFAULT '0' COMMENT 'Client side holiday id',
  `description` varchar(255) DEFAULT NULL COMMENT 'Description of the event displayed in console',
  `world_event` tinyint(3) unsigned NOT NULL DEFAULT '0' COMMENT '0 if normal event, 1 if world event',
  `announce` tinyint(3) unsigned DEFAULT '2' COMMENT '0 dont announce, 1 announce, 2 value from config',
  PRIMARY KEY (`eventEntry`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8

Error occured at:2017-03-31 17:33:23
Line no.:1642427
Error Code: 1067 - Invalid default value for 'start_time'


i change default time to : 0000-00-00 00:00:00  and work fine

**Comment by gpn39f** [comment]
a time value of 0000-00-00 00:00:00 should make error.. so i think:

check: youre mysql server should have version 5.5.x.. NO NEWER VERSION...
ckeck: in youre  mysql server configuration you has to set value of buffer to 32M

**Comment by Nephelion** [comment]
i have new  problem open new issue ?
i have error for table BroadcastText . this table not exist but i found it on clean world [creat world database]
now this table is empty and i not found any import data
my errors like : 
ossipText (Id: 1) in table `npc_text` has non-existing or incompatible BroadcastTextID0 50429.
GossipText (Id: 68) in table `npc_text` has non-existing or incompatible BroadcastTextID0 50429.
GossipText (Id: 197) in table `npc_text` has non-existing or incompatible BroadcastTextID0 7590.

--CreatureTextMgr: Entry 55729, Group 0, Id 6 in table `creature_texts` has non-existing or incompatible BroadcastTextId 16324.
CreatureTextMgr: Entry 54485, Group 0, Id 5 in table `creature_texts` has non-existing or incompatible BroadcastTextId 53453.
CreatureTextMgr: Entry 55729, Group 0, Id 0 in table `creature_texts` has non-existing or incompatible BroadcastTextId 16318.
CreatureTextMgr: Entry 54485, Group 0, Id 4 in table `creature_texts` has non-existing or incompatible BroadcastTextId 53452.
 and more than 70.000 line like this
------i need broadcast_text  data

now i testing source [quest and etc]
-----and some crash when Quest give or take [now dont have log in realease with debug info]

**Comment by gpn39f** [comment]
you have to read more the text in snapshoot..
in folder persist data you find this two tables..

**Comment by gpn39f** [comment]
maybe its better you write it in our forum..
http://gpn39f.spdns.de/forum/

**Comment by Nephelion** [comment]
When i Use 0000-00-00 00:00:00 for Game event start time more event Buged
what i do is better ? any number like 1970 ....  import error

**Comment by gpn39f** [comment]
this question you have to ask unix or mysql..
if you look at google, the unix time starting 1970.. 
but what is a valid month?? jan=1
what is a valid day?? first jan=1970/01/01
and same question about time..
what is a valid time?? unix time starting 1970 at "first" second in first hour, not 0 second!!!!
and you have am/pm
so a valid time is : 01/01/1970 @ 12:00:01am (UTC)
or
1970-01-01 01:00:01
